test/e2e/probe_whitelist_test.go

```diff
@@ -36,6 +36,7 @@ import (
 //policy is present in test/config/security/policy.yaml
 //apply policy before running this test
 func TestProbeWhitelist(t *testing.T) {
+	t.Skip("RequestAuthentication dropped the regex support. We need to rewrite this test. See https://github.com/istio/istio/issues/16585")
```

Comment:
Why are we skipping tests for Istio?  If this is an istio specific test, it should go somewhere other than `test/e2e`, and if it isn't then this reduces the coverage for other ingress providers as well.